swap/src/config.rs

```diff
@@ -49,27 +49,24 @@ pub struct Monero {
     pub wallet_rpc_url: Url,
 }
 
-#[derive(thiserror::Error, Debug)]
-pub enum Error {
-    #[error("config not initialized")]
-    ConfigNotInitialized,
-    #[error("other error")]
-    Other(#[from] anyhow::Error),
-}
+#[derive(thiserror::Error, Debug, Clone, Copy)]
+#[error("config not initialized")]
+pub struct ConfigNotInitialized {}
```

Comment:
Do you mean that the configuration is missing?

Reply:
It is not initialized. The current process is: Upon initial startup the user is queried for the RPC urls and wallet name (bitcoin only as long as we are using bitcoind's wallet) and the config is written automatically. 

We can rename it to `ConfigMissing`, but it is only used for triggering the `initial_setup` at the moment, so I think the name `ConfigNotInitialized` expresses exactly what it is.